docs/DeveloperGuide.md

```diff
@@ -1,33 +1,230 @@
-# Developer Guide
+# ActiveEdge Developer Guide 🥦🏃‍♂️
 
```

Comment:
Overall, the Developer Guide looks neat and well formatted. However, the link 'design and implementation' under the table of contents are broken.